style.css

```diff
@@ -0,0 +1,247 @@
+*{
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+input, button {
+  font-family: inherit;
+}
+
+body {
+  background: url(img/Background.png) no-repeat;
+  background-size: cover;
+  font-family: 'Roboto', sans-serif;
+  margin: 0 auto;
+  min-width: 320px; 
+  max-width: 1440px;
+  /*min-height: 1024px;*/
+}
+
+.main {
+  max-width: 90%;
+  margin: auto;
+}
+
+.container,
+.three/*,
+.six,
+.ten */{
+  min-height: 1024px;
+}
+
+.level {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 4.8%;
+}
+
+.level__choice {
+  font: normal bold 57px/ 67px 'Roboto';
+  color: #FFFFFF;
+  padding: 10% 0 0.6% 2.4%;
+}
+
+.level__difficulty {
+  font: normal bold 19px/ 21px 'Roboto';
+  color: #FFFFFF;
+  padding: 0 0 0 2.5%;
+}
+
+.game input {
+  outline: none;
+  background: none;
+  color: #FFFFFF;
+  font: normal bold 50px/ 59px 'Roboto';
+  align-self: flex-start;
+  margin-left: 6%;
+  cursor: pointer;
+}
+
+.game input:hover {
+  -webkit-transform: scale(1.1);
+  -ms-transform: scale(1.1);
+  transform: scale(1.1);
+}
+
+.game {
+  display: flex;
+  flex-direction: column;
+}
+
+.game label {
+  position: relative;
+}
+
+/*квадратик */
+
+.simple-level:before,
+.medium-level:hover:before {
+  background: url(img/square.png) no-repeat;
+  background-size: cover;
+  content: "";
+  width: 25px;
+  height: 25px;
+  position: absolute;
+  top: 19.5%;
+  right: 95.7%;
+}
+
+.game__simple {
+  margin-bottom: 1.8%;
+}
+
+.game__medium {
+   margin-bottom: 1.8%;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/cardBagg/pullRequest?prid=5838315)